RedfishEventListener_v1.py

```diff
@@ -24,7 +24,7 @@
 standard_out.setLevel(logging.INFO)
 my_logger.addHandler(standard_out)
 
-tool_version = '1.1.2'
+tool_version = '1.1.3'
```

Comment:
No need to add this; our release workflow fills in the changelog when we press the "release" button.